### PR TITLE
Remove unneeded file.

### PR DESCRIPTION
**Pull request by mikeller**
Fixes #527.

The link to this file was removed in https://github.com/betaflight/blackbox-log-viewer/commit/9ff4cf4f0ee732bc59eb265766bb75c1d199bba9#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L30 (after having been added in https://github.com/betaflight/blackbox-log-viewer/commit/4ec112eeccf51e0fa94642378e298690beba1f78).

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
